terraform/app/modules/cloudfront/main.tf

```diff
@@ -30,18 +30,11 @@ resource "aws_cloudfront_distribution" "default" {
   aliases = local.cloudfront_aliases
 
   default_cache_behavior {
-    allowed_methods  = ["DELETE", "GET", "HEAD", "OPTIONS", "PATCH", "POST", "PUT"]
-    cached_methods   = ["GET", "HEAD"]
-    target_origin_id = "${var.service_name}-${var.environment}-default-origin"
-
-    forwarded_values {
-      query_string = true
-      headers      = var.default_header_list
-
-      cookies {
-        forward = "all"
-      }
-    }
+    allowed_methods          = ["DELETE", "GET", "HEAD", "OPTIONS", "PATCH", "POST", "PUT"]
```

Comment:
Could you move the `default_cache_behavior` block after the other `ordered_cache_behavior` blocks? This would show the processing order explicitly and would match the UI visually